pkg/wsman/amt/auditlog/decoder.go

```diff
@@ -186,90 +187,103 @@ func convertToAuditLogResult(auditlogdata []string) []AuditLogRecord {
 		exlen := []byte(decodedEventRecordStr[ptr : ptr+1])[0]
 		ptr++
 		auditLogRecord.Ex = decodedEventRecordStr[ptr : ptr+int(exlen)]
-		// auditLogRecord.ExStr = GetAuditLogExtendedDataString((auditLogRecord.AuditAppID*100)+auditLogRecord.EventID, auditLogRecord.Ex)
+		auditLogRecord.ExStr = GetAuditLogExtendedDataString(AuditEventId((auditLogRecord.AuditAppID*100)+auditLogRecord.EventID), auditLogRecord.Ex)
 
 		records = append([]AuditLogRecord{auditLogRecord}, records...)
 	}
 
 	return records
 }
 
+type AuditEventId int
```

Comment:
Do we need this type? What can't we use int directly?

Reply:
good catch!  updated